app/src/main/java/com/framgia/soundclound/screen/local/LocalFragment.java

```diff
@@ -46,6 +48,8 @@ public void onRequestPermissionsResult(int requestCode,
         super.onRequestPermissionsResult(requestCode, permissions, grantResults);
         if (requestCode == Constant.REQUEST_READ_STORAGE) {
             // TODO: 1/11/2018
+            mViewModel.getTrackLocalAdapter().addData(new TrackRepository
+                    (TrackRemoteDataSource.getInstance()).getLocalTrack());
```

Comment:
ERROR: 'operator new lparen' have incorrect indentation level 20, expected level should be 12.